src/components/date-picker/date-picker.tsx

```diff
@@ -160,7 +160,11 @@ export class DatePicker {
     }
 
     public componentWillLoad() {
-        this.useNative = isIOSDevice() || isAndroidDevice();
+        this.useNative =
+            isIOSDevice() ||
+            isAndroidDevice() ||
+            this.disabled ||
+            this.readonly;
```

Comment:
Hmm… I'm not sure this is the right way to go about it. This removes the formatting for a bunch of the types…

Isn't it possible to just prevent the flatpickr menu from opening instead? I can have a look at it if you'd like me to.

Reply:
![image](https://user-images.githubusercontent.com/154725/122198647-9cbecc00-ce99-11eb-888f-07c15c321be4.png)

Reply:
> Hmm… I'm not sure this is the right way to go about it. This removes the formatting for a bunch of the types…
> 
> Isn't it possible to just prevent the flatpickr menu from opening instead? I can have a look at it if you'd like me to.

This was my quick fix for this. I agree that would be better but I couldn't find a good way to solve it. Feel free to look in to it if you want!

Aaah, the quarter picker with readonly only works in the web client then as that's were we're setting the format 🤦

Reply:
Another quick fix could be to set type=text for all pickers and just render the date in a normal limel-input-field

Reply:
I'll have a look 👍

Reply:
I've pushed an alternative version. Please have a look 🙂

Reply:
Oh, btw, it's three separate commits. The first one is just some boy-scouting, the second is the fix, and the third is just adding the composite example to the docs. So I'd suggest focusing on the middle commit 🙂